package.json

```diff
@@ -30,11 +30,11 @@
   "homepage": "https://wwebjs.dev/",
   "dependencies": {
     "@pedroslopez/moduleraid": "^5.0.2",
-    "fluent-ffmpeg": "^2.1.2",
-    "jsqr": "^1.3.1",
+    "fluent-ffmpeg": "2.1.2",
+    "jsqr": "1.3.1",
```

Comment:
```suggestion
```
@tuyuribr 
`jsqr` already is not in use, should we get rid of it? 
Was added in [#26](https://github.com/pedroslopez/whatsapp-web.js/commit/bea1ebf48035f52b71b145214eac0dfd6576b113) and then its usage was removed in [#68](https://github.com/pedroslopez/whatsapp-web.js/commit/5c9e76e23f52418eaa3062e33194eb916c9565ef)